modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/tsconfig.json

```diff
@@ -0,0 +1,54 @@
+{
+	"compilerOptions": {
```

Comment:
In our previous tsconfig we had something like

```
"@generated": "3c395c855c2853aafc34ba33f50d97de419e8cef",
	"@overrides": {
	},
	"@readonly": "** AUTO-GENERATED: DO NOT EDIT OUTSIDE @overrides **",
```

Is something like that necessary again? I especially like the "auto-generated do not edit" portion, as this ensures consistency in the future of peoples configs.

Reply:
One other note is that by using the hash in "@generated" allows us to not keep re-building the json, especially when there may be issues of SF. Currently gradle SFs json files, so we may get a headache with re-genning and colliding with SF rules, see below for example.

<img width="950" alt="Screenshot 2024-05-30 at 11 19 10 AM" src="https://github.com/liferay-frontend/liferay-portal/assets/6843530/9a8f4fb3-1a11-4d1d-bbb1-cd1a3753dd81">

Reply:
That's true. I was assuming `generate:tsconfig` was going to be run manually because initially it took too long. However I've optimized it now and it's quite fast so it makes sense to run it always, prior to checking types.

Reply:
Will address this in https://liferay.atlassian.net/browse/LPD-27157